index.js

```diff
@@ -7,7 +7,6 @@ const URL = require('url').URL
 const { https } = require('follow-redirects')
 const AdmZip = require('adm-zip')
 const HttpsProxyAgent = require('https-proxy-agent')
-var proxy = process.env.http_proxy || 'http://168.63.76.32:3128';
```

Comment:
I was surprised when I saw this random IP address, but this variable is not actually being used and I removed it.

Reply:
Weird indeed, it looks to be copy pasta from the `https-proxy-agent` docs, though I don't know why they would include a hardcoded IP address there either.

Reply:
I don't recall if I intentionally added this or just copy-pasta'd it or what. Either way, shame on me, I should have commented what and why.